### PR TITLE
fix-padding-input-range-chrome

### PR DESCRIPTION
**Pull request by benschac**
Fixes https://github.com/sveltejs/svelte/issues/5234

Image 2020-08-03 at 8 46 09 PM
to this
Image 2020-08-03 at 10 12 52 PM

### CODE REVIEW

**Comment by caroso1222** [comment]
Can you please put a `Fixes https://github.com/sveltejs/svelte/issues/5234` in the description?

**Comment by benschac** [comment]
updated!

File: public/global.css [hunk 1]
```diff
@@ -32,6 +32,8 @@ label {
 input, button, select, textarea {
 	font-family: inherit;
 	font-size: inherit;
+	/* Fix Padding In Chrome */
```
Comment (antony) [inline public/global.css:35]: Not sure the comment is necessary?

Comment (Conduitry) [reply public/global.css:35]: +1 to this. None of the other styles have comments.

Comment (benschac) [reply public/global.css]: removed. ty for the feedback

**Pull request merged**